# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1282414706 25200
#      Sat Aug 21 11:18:26 2010 -0700
# Node ID 69a2e9ccb2fb53cf02d9a1be55640ac90e892722
# Parent  9f0f1a89c7048b6731f5cd258c78f82fedabb552
NEWS: Add randi to list of new functions

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2010-08-21  Rik <octave@nomad.inbox5.com>
+
+	* NEWS: Add randi to list of new functions
+
 2010-08-09  John Swensen <jpswensen@gmail.com>
 
 	* mkoctfile.in: Add ability to pass -pthread through to linker.
 
 2010-08-13  Jaroslav Hajek  <highegg@gmail.com>
 
 	* NEWS: Update.
 
diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -311,23 +311,23 @@ Summary of important user-visible change
   ** Two new formats are available for displaying numbers:
      format short eng
      format long eng
      Both display numbers in engineering notation, i.e., mantissa + exponent
      where the exponent is a multiple of 3.
 
   ** The following new functions were added:
 
-      accumdim     erfcx        luupdate     ppder        sizemax
-      bitpack      fileread     merge        ppint        strread
-      bitunpack    fminbnd      nfields      ppjumps      textread
-      blkmm        fskipl       nth_element  pqpnonneg    whitebg
-      cbrt         ifelse       onCleanup    repelems
-      chop         ishermitian  pbaspect     reset
-      daspect      isindex      powerset     rsf2csf
+      accumdim     erfcx        luupdate     ppder        rsf2csf
+      bitpack      fileread     merge        ppint        sizemax
+      bitunpack    fminbnd      nfields      ppjumps      strread
+      blkmm        fskipl       nth_element  pqpnonneg    textread
+      cbrt         ifelse       onCleanup    randi        whitebg
+      chop         ishermitian  pbaspect     repelems     
+      daspect      isindex      powerset     reset
   
   ** The behavior of struct assignments to non-struct values has been changed.
      Previously, it was possible to overwrite an arbitrary value:
       
         a = 1;
         a.x = 2;
 
      This is no longer possible unless a is an empty matrix or cell array.
